src/applications/static-pages/facilities/FacilityPhone.jsx

```diff
@@ -15,31 +15,38 @@ export default class FacilityPhone extends React.Component {
 
     return (
       <div className="vads-u-margin-bottom--0">
-        <div className="main-phone vads-u-margin-bottom--1">
-          <strong>Main phone: </strong>
-          <a
-            href={`tel:${this.props.facility.attributes.phone.main.replace(
-              /[ ]?x/,
-              '',
-            )}`}
-          >
-            {this.props.facility.attributes.phone.main.replace(/[ ]?x/, '')}
-          </a>
-        </div>
-        <div className="mental-health-clinic-phone">
-          <strong>Mental health clinic: </strong>
-          <a
-            href={`tel:${this.props.facility.attributes.phone.mentalHealthClinic.replace(
-              /[ ]?x/,
-              '',
-            )}`}
-          >
-            {this.props.facility.attributes.phone.mentalHealthClinic.replace(
-              /[ ]?x/,
-              '',
-            )}
-          </a>
-        </div>
+        {this.props.facility.attributes.phone.main && (
+          <div className="main-phone vads-u-margin-bottom--1">
+            <strong>Main phone: </strong>
+            <a
+              href={`tel:${
+                // this replaces an "x" that appears from the api data with a space
+                this.props.facility.attributes.phone.main.replace(/[ ]?x/, '')
```

Comment:
I realize this existed before this PR, but I think this formatting should be relegated to a helper function. I guess `facilityUtilities.js` in the same directory would the be home for it. Also, does it replace with a space, or does it just remove the x completely?

```
// This removes (?) an "x" that appears from the API data.
const cleanPhoneNumber = number => number.replace(/[ ]?x/, '');
```

You can also save an extra replace each time you use it (also in the other component):

```
const { phone } = this.props.facility.attributes;
const mainPhone = cleanPhoneNumber(phone.main);
const mentalHealthClinicPhone = cleanPhoneNumber(phone.mentalHealthClinic);
// Use formatted numbers in the markup below.
```

Reply:
i like the suggestion and implemented :smile:  thank you!